src/app/services/auth/auth.service.ts

```diff
@@ -37,9 +37,10 @@ export class AuthService {
    */
   readonly tokenRegenerationTimeMillis = 290 * 1000;
 
-  private readonly latestTokenGenerated$ = new ReplaySubject<string>(1);
+  private readonly latestTokenGenerated$ = new Subject<string>();
+
   get authToken$(): Observable<string> {
-    return this.latestTokenGenerated$.asObservable();
+    return of(this.token);
```

Comment:
The `ReplaySubject` was used for if you stay on this shell page for longer than 5 minutes without doing anything, the token needs to be refreshed. This will only emit the token once when it's used and subsequently, the token won't be emitted and shell will be expired. We have to come up with another solution.

Reply:
You mean we don't need to see this modal?
<img width="474" alt="Screenshot 2023-04-05 at 13 00 39" src="https://user-images.githubusercontent.com/22980553/230048691-6ab13d80-f461-488d-945e-0cf28c15994e.png">

Reply:
@RehanY147